validation/src/main/resources/application.conf

```diff
@@ -17,12 +17,20 @@ hmda.validation.quality {
     min-income-for-high-loan=9
     min-income-for-high-loan=${?Q024_MIN_INCOME}
   }
+  Q025 {
+    loan.amount=10
+    loan.amount=${?Q025_LOAN_AMOUNT}
+  }
   Q036 {
     loan.amount=150
     loan.amount=${?Q036_LOAN_AMOUNT}
   }
   Q037 {
-    loan.amount=250
-    loan.amount=${?Q037_LOAN_AMOUNT}
+    loan.amount = 250
+    loan.amount = ${?Q037_LOAN_AMOUNT}
+  }
+  Q038 {
+    loan.amount = 100
+    loan.amount = ${?Q035_LOAN_AMOUNT}
```

Comment:
Is that something we can configure? (Or, do we wanna standardize on doing whatever it does, as long as it's not too icky?) Not a blocker for this PR, but might help avoid this kind of noise.